qiskit_nature/transformers/particle_hole_transformer.py

```diff
@@ -12,13 +12,30 @@
 
 """The Particle/Hole Transformer interface."""
 
-from qiskit_nature.drivers import QMolecule
-from .base_transformer import BaseTransformer
+from typing import Optional
+import numpy as np
+import copy
+
+from .. import QiskitNatureError
 
+from .base_transformer import BaseTransformer
+from ..drivers import QMolecule
 
 class ParticleHoleTransformer(BaseTransformer):
     """The Particle/Hole transformer."""
 
+    def __init__(self, num_electrons: int, num_orbitals: int, num_alpha: int):
+        """Initializes a transformer which can reduce a `QMolecule` to a configured active space.
+
```

Comment:
Given the transformer is acting directly on a QMolecule, that contains all this information, it should not be necessary to pass this information in since it should all be there in the qmolecule. Indeed expecting the user to figure this when it may be in a series of tranforms that are affecting num electrons etc seems less than desirable.

Reply:
You are right. I am fixing this now.